### PR TITLE
Sampleview all terms

### PR DESCRIPTION
**Pull request by airenzp**
## Description

Added survival and conditional term values to the sample view.

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: server/src/termdb.server.init.js [hunk 1]
```diff
@@ -471,9 +471,28 @@ export function server_init_db_queries(ds) {
 		union all  
 		select term_id, value 
 		from anno_integer 
-		where sample=? ${termClause} ) join terms on terms.id = term_id`
+		where sample=? ${termClause}
+		union all 
+		select term_id, value
```
Comment (xzhou82) [inline server/src/termdb.server.init.js:476]: should also select `min_years_to_event` and display it on client

Comment (airenzp) [reply server/src/termdb.server.init.js]: Should be fixed now

File: client/plots/sampleView.js [hunk 2]
```diff
@@ -129,11 +129,20 @@ class SampleView extends MassDict {
...
 		let value = this.sampleData[term.id]?.value
-
 		if (value == null || value == undefined) return null
 		if (term.type == 'float' || term.type == 'integer')
 			return value % 1 == 0 ? value.toString() : value.toFixed(2).toString()
 		if (term.type == 'categorical') return term.values[value].label || term.values[value].key
+		if (term.type == 'condition') {
+			let [years, status] = value.split(' ')
+			status = term.values[status].label || term.values[status].key
+			return `${status} after ${Number(years).toFixed(1)} years`
```
Comment (xzhou82) [inline client/plots/sampleView.js:139]: please replace wording as `Max grade: xx, time-to-event: yy years`

Comment (airenzp) [reply client/plots/sampleView.js:139]: Updated now, survival terms are ok?

Comment (xzhou82) [reply client/plots/sampleView.js:139]: hope that's okay. we can always change later. thanks

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**